llvm/lib/IR/RepoTicket.cpp

```diff
@@ -116,7 +116,39 @@ static GODigestState accumulateGOFinalDigestOrInitialDigestAndGetGODigestState(
   return accumulateGOInitialDigestAndGetGODigestState(GO, GOHash, GOIMap);
 }
 
-// Update the GO's hash value by adding the hash of its dependents.
+// Loop through the Contributions and add the the G's initial digest to the
+// GOHash.
+static bool updateDigestUseContributions(MD5 &GOHash,
+                                         const GOVec &Contributions,
+                                         GOInfoMap &GOIMap) {
+  for (const GlobalObject *const G : Contributions) {
+    GOHash.update(GOIMap[G].InitialDigest.Bytes);
+  }
+  return !Contributions.empty();
+}
```

Comment:
I think that this needs to say somewhere that the return value indicates that the hash has changed. The other function has a variable so doesn't need it so much. This could either be in the comment for the whole function or a comment before the return line.

Reply:
Thanks, a comment is added.